pkg/compiler/codegen.go

```diff
@@ -1138,8 +1138,12 @@ func (c *codegen) getByteArray(expr ast.Expr) []byte {
 			val := constant.StringVal(tv.Value)
 			return []byte(val)
 		}
-
 		return nil
+	case *ast.Ident:
```

Comment:
I don't think it's supposed to work with `ast.Ident` (`getByteArray returns byte array value from constant expr`).